src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -339,7 +339,14 @@ public Workspace getWorkspace() {
 
     public void useCustomWorkspace(String ws) {
         ensureConfigPage();
-        clickButton("Advanced...");
+
+        // There may be multiple "Advanced..." buttons visible on the job config page, and there's no easy way to identify
+        // which one is the Advanced Project Options one, so let's just hit all of them.
+        for (WebElement advancedButton : all(by.button("Advanced..."))) {
+            if (advancedButton.isDisplayed()) {
+                advancedButton.click();
+            }
```

Comment:
Thought about that - I think I'm going to do a general refactoring of common code etc in the ATH in the next few weeks and will do this then.